contracts/Registry.sol

```diff
@@ -29,6 +27,7 @@ contract Registry {
     function addNewContract(bytes32 _id, address _contractAddress) external onlyGovernance {
         require(!modules[_id].exists, 'Registry::addNewContract: Entry already exists.');
```

Comment:
What is the need for an "exists" field on the struct? Can an Entry be active and not exist at the same time?